interop/xds/server/server.go

```diff
@@ -53,6 +58,16 @@ var (
 	logger = grpclog.Component("interop")
 )
 
+const (
+	rpcBehaviorMdKey  = "rpc-behavior"
+	rpcAttemptsMdKey  = "grpc-previous-rpc-attempts"
```

Comment:
Nit: Consider renaming this as `grpcPreviousRPCAttempts` or `grpcPreviousRPCAttemptsKey` or `grpcPreviousRPCAttemptsMDKey`, to more closely match the actual key name.

Also, it needs to be `MD` and not `Md` because of https://google.github.io/styleguide/go/decisions#initialisms.